commit e0948518f6eedad85c7705fb3963cfdda3627d8f
Author: Reto Gantenbein <reto.gantenbein@linuxmonk.ch>
Date:   Thu Oct 22 00:19:31 2020 +0200

    Always skip init/oom test
    
    The RPM package build in COPR is being executed on a tmpfs which
    always results in a failure of this test:
    
    init/oom
      heap-fault-delay=1, heap-fault-repeat=1                   [ ERROR ]
    Error: test/integration/test_uv_init.c:156: assertion failed: _rv == 1 (0 == 1)
    
    Instead only for certain architectures we always skip this test.
    For more informations see https://github.com/ganto/copr-lxc4/issues/8

diff --git a/test/integration/test_uv_init.c b/test/integration/test_uv_init.c
index 8c9b5f7..57db7f3 100644
--- a/test/integration/test_uv_init.c
+++ b/test/integration/test_uv_init.c
@@ -144,10 +144,8 @@ static MunitParameterEnum oomParams[] = {
 TEST(init, oom, setUp, tearDown, 0, oomParams)
 {
     struct fixture *f = data;
-#if defined(__i686__)
     /* XXX: tmpfs seems to not support O_DIRECT */
     return MUNIT_SKIP;
-#endif
 #if defined(__powerpc64__)
     /* XXX: fails on ppc64el */
     return MUNIT_SKIP;
